study-group/README.md

```diff
@@ -0,0 +1,119 @@
+# Toast Study Group
+
+This document compares the capabilities of various popular toast libraries.
+
+## Background
+A toast is a UI element that provides valuable feedback to a user about the actions they perform on the web or their mobile device. This feedback can be core to the user experience of many applications, and many accomplished developers have written many libraries to address this need. The aim of this study is to assess which needs the existing solutions meet, which services they provide in common with each other, and what the levels of granularity are available in toast APIs.
```

Comment:
I'm a big fan of semantic line breaks: https://sembr.org/

It would be great if you could use them. The reason I like them (that doesn't seem to be listed on that page) is that diffs are generally much easier to read (vs 1 line per paragraph or vs wrapping to 80 cols).

Reply:
Got it, I'll change what I have now and follow this going forward, thanks